### ISSUE TITLE
d3m.primitives.tods.timeseries_processing.transformation

### ISSUE THREAD

**Issue by danidadon** [datamllab/tods#59]
As of a week or so ago, I was able to run everything fine including the AutoML example with my dataset. However, I believe there were some updates made and now I am getting this error when I changed nothing to the code. Any insight into this is appreciated.

Thank you.

<img width="935" alt="Screenshot 2022-04-19 140450" src="https://user-images.githubusercontent.com/104027289/164067570-d2365622-9f0e-40a7-85e2-1537dcff6b4a.png">

**Comment by mdforney** [comment]
You are right. I was in dev branch. This initial code works like previous version. Thank you.

import warnings
warnings.filterwarnings("ignore")

!pip install -e git+https://github.com/datamllab/tods.git#egg=tods 
exit()

**Comment by mdforney** [comment]
With the previous version I am wondering how to remove a particular primitive. For example, if I wanted the AutoML searcher to exclude the autoencoder primitive what would be the suggestion.

Thank you.
